build.gradle

```diff
@@ -21,7 +21,7 @@ targetCompatibility = 1.8
 dependencies {
     compile  "org.embulk:embulk-core:0.9.20"
     provided "org.embulk:embulk-core:0.9.20"
-    compile group: 'com.cybozu.kintone', name: 'kintone-sdk', version: '0.4.0'
+    compile group: 'com.cybozu.kintone', name: 'kintone-sdk', version: '0.6.2'
```

Comment:
Most of embulk users are still using 0.9 which depends on Java 8. Is this work embulk 0.9.23?

Reply:
Version 0.9.23 worked fine with the following warning.

```
[WARN] Embulk does not guarantee running with Java 9.
```

Reply:
Thank you for trying. IMO, the stable transfer is the most important in terms of ETL to rescue engineers from nightmare:) We should stick embulk dependencies. For the plan of upgrading Java 9+ of embulk, please see this material. https://www.embulk.org/assets/files/2020-07-09-meetup.pdf